contracts/token/ERC20/extensions/ERC1363.sol

```diff
@@ -30,16 +30,23 @@ abstract contract ERC1363 is ERC20, ERC165, IERC1363 {
 
     /**
      * @dev Indicates a failure within the {transfer} part of a transferAndCall operation.
+     * @param receiver The address to which tokens are being transferred.
```

Comment:
we are using a slightly different natspec comment for `receivers`, as shown [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/0a2dc05bdf78144867ee824adb63d742abdb9be3/contracts/token/ERC20/extensions/ERC1363.sol#L21) and [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/06449fe7bcfaafe43abdfc2b28fb6bb189e6ade5/contracts/interfaces/draft-IERC6093.sol#L26).

```suggestion
     * @param receiver Address to which tokens are being transferred.
```

Reply:
Yeah sounds less repetitive.